packages/doc-ai-react/src/App.css

```
@@ -1,44 +1,1190 @@
+  margin-bottom: 2rem;
 }
 
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+.construction {
+  font-size: 3rem;
 }
 
-@keyframes logo-spin {
+/* Animations */
+@keyframes fadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)